docs/UserGuide.adoc

```diff
@@ -250,6 +252,7 @@ Assigns the student with matric no. `A0171234B` as a TA.
 * `module assign e/e0191234@u.nus.edu` +
 Assigns the student with email `e0191234@u.nus.edu` as a TA.
 
+=======
```

Comment:
This particular line seems to have cut off the subsequent sections from the table of contents